src/lime_qt/configuration/configure_system.cpp

```diff
@@ -236,6 +238,32 @@ ConfigureSystem::ConfigureSystem(Core::System& system_, QWidget* parent)
             &ConfigureSystem::RefreshConsoleID);
     connect(ui->button_start_download, &QPushButton::clicked, this,
             &ConfigureSystem::DownloadFromNUS);
+
+    connect(ui->button_secure_info, &QPushButton::clicked, this, [this] {
+        ui->button_secure_info->setEnabled(false);
```

Comment:
Better safe than sorry? There are a lot of different windowing systems out there (on the various *nix's especially) and who knows if they all behave the same in terms of window focus. In terms of defensive coding, I actually like this approach and feel all the buttons that share the same kind of functionality should be this way too. But that's just my opinion; maybe it's an old QT holdover mindset that isn't an issue anymore but is just a habit; Citra used to run on QT5 and maybe still being able to click on the button despite a file dialog being open was an issue there, for example.

Reply:
You have a point 

But *i think* citra uses XDG_DESKTOP PORTALS for the file dialog on linux / bsd, where this is the most likely to accur 

Afaik, that should handle properly displaying and focusing the file dialog like no matter the DE / tiling WM 
(which are all up-to-date environments , only should be an issue if you use RHEL or Debian 11)

Anyway, I'll let OpenSauce decide on this one since he's more comfortable with unix like systems than me

Reply:
To be fair, not all window managers on *nix respect XDG specs. For example, dwm doesn't by default on purpose, and others like i3 and Openbox only have partial support. So maybe QT best practice is what (I assume) PabloMK7 did and assume nothing about what the program is running on and code appropriately. But I'm not a QT expert.

Reply:
Beats be why he decided to do this here and seemingly didn't change the behaviour anywhere else, but I don't think there's any issue with this. We should probably update the behaviour elsewhere in a separate PR for consistency though.

Reply:
Aight ill mark this as resolved fn